File: TestExecution/TestAdapters/BUnit.TestAdapter/VsTestDiscoverer.fs
namespace BUnit.TestAdapter

open Microsoft.VisualStudio.TestPlatform.ObjectModel
open Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter

type VsTestDiscoverer() = 
    let rt = Runtime.Class1()
    interface ITestDiscoverer with
        member __.DiscoverTests(_ : System.Collections.Generic.IEnumerable<string>, _ : IDiscoveryContext, 
                                _ : Logging.IMessageLogger, _ : ITestCaseDiscoverySink) : unit = 
            failwith "Not implemented yet"

File: TestExecution/TestAdapters/AUnit.TestAdapter/VsTestDiscoverer.cs
using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
using System;
using System.Collections.Generic;

namespace AUnit.TestAdapter
{
    public class VsDiscoverer : ITestDiscoverer
    {
        private Runtime.Class1 rt = new Runtime.Class1();

        public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discoveryContext, IMessageLogger logger, ITestCaseDiscoverySink discoverySink)
        {
            throw new NotImplementedException();
        }
    }
}
